kobo/apps/user_reports/migrations/0002_reset_mv_for_djstripe_update.py

```
@@ -0,0 +1,22 @@
+def reset_lrm_0019(apps, schema_editor):
+
+    LongRunningMigration = apps.get_model(
+        'long_running_migrations', 'LongRunningMigration'
+    )
+    LongRunningMigration.objects.filter(name__startswith='0019').update(
+        status='created'
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [('user_reports', '0001_squashed_old_migrations')]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)